File: samples/chirper/front-end/app/controllers/Application.java
package controllers;

import play.mvc.Controller;
import play.mvc.Result;

public class Application extends Controller {

  public Result index() {
    return ok(views.html.index.render());
  }

  public Result userStream(String userId) {
    return ok(views.html.index.render());
  }

  public Result circuitBreaker() {
    return ok(views.html.circuitbreaker.render());
  }

}

File: dev/sbt-plugin/src/sbt-test/sbt-plugin/distribution/playProj/app/controllers/Application.scala
/*
 * Copyright (C) Lightbend Inc. <https://www.lightbend.com>
 */

package controllers

import play.api._
import play.api.mvc._
import scala.collection.JavaConverters._

import javax.inject.Inject

class Application @Inject()(
  env: Environment,
  configuration: Configuration,
  val controllerComponents: ControllerComponents
) extends BaseController {

  def index = Action {
    Ok(views.html.index("Your new application is ready."))
  }

  def config = Action {
    Ok(configuration.underlying.getString("some.config"))
  }

  def count = Action {
    val num = env.resource("application.conf").toSeq.size
    Ok(num.toString)
  }
}
